### PR TITLE
Register

### PR DESCRIPTION
**Pull request by vanhauna**

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name','birthday', 'email', 'password','address','phone','card_id','role',
```
Comment (sonnguyen8x) [inline app/User.php]: conventsion 
'name', 'birthday',  'email',  'password' ,'address' ,'phone' ,'card_id' ,'role',

File: resources/views/admin/add.blade.php [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+add
```
Comment (sonnguyen8x) [inline resources/views/admin/add.blade.php]: dòng trắng cuối file

File: resources/views/admin/delete.blade.php [hunk 3]
```diff
@@ -0,0 +1 @@
+delete
```
Comment (sonnguyen8x) [inline resources/views/admin/delete.blade.php]: convention

File: app/User.php (hunk 1 above)
Comment (vanhauna) [reply app/User.php]: done

File: resources/views/admin/add.blade.php (hunk 2 above)
Comment (vanhauna) [reply resources/views/admin/add.blade.php]: done

File: resources/views/admin/delete.blade.php (hunk 3 above)
Comment (vanhauna) [reply resources/views/admin/delete.blade.php]: done

File: database/migrations/2019_06_04_120056_create_set_default_value_of_role_table.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('users');
```
Comment (sonnguyen8x) [inline database/migrations/2019_06_04_120056_create_set_default_value_of_role_table.php]: không phải drop users

File: app/User.php [hunk 5]
```diff
@@ -15,7 +15,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name','birthday','email','password','address','phone','card_id','role',
```
Comment (sonnguyen8x) [inline app/User.php]: conventions

File: database/migrations/2019_06_04_120056_create_set_default_value_of_role_table.php (hunk 4 above)
Comment (vanhauna) [reply database/migrations/2019_06_04_120056_create_set_default_value_of_role_table.php]: fix done

File: app/User.php (hunk 5 above)
Comment (vanhauna) [reply app/User.php]: fix done

**Pull request merged**